docs/_tutorials/ja_getting_started.md

```diff
@@ -161,7 +161,8 @@ const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   socketMode: true,
-  appToken: process.env.SLACK_APP_TOKEN
+  appToken: process.env.SLACK_APP_TOKEN,
+  port: process.env.PORT || 3000 // ソケットモードはポートでリッスンしませんが、アプリがOAuthに応答するようにしたい場合は、ポートでリッスンする必要があります。
```

Comment:
@seratch please excuse my poor translation (using Google Translate) - could you take a look at this and make sure it is not embarrassingly bad? Thank you! 🙏

Reply:
@wongjas may also be able to assist here!